### PR TITLE
Reduce image size - #98

### PR DESCRIPTION
**Pull request by PhilippeChepy**
- Reduced image size from 550,5 MB to 386 MB.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,74 +1,71 @@
 FROM ubuntu:14.04
```
Comment (jmparra) [inline Dockerfile]: maybe move to [Debian:8.3](https://imagelayers.io/?images=ubuntu:14.04,debian:8.3) can help

Comment (PhilippeChepy) [inline Dockerfile]: I have tested with Debian:jessie and curiously size of image increased. But it produce a working image as well.

Comment (jmparra) [inline Dockerfile]: works for me, but i use other [cleanup](https://hub.docker.com/r/cgswong/min-jessie/) process
```
apt-get -y clean && apt-get -y autoclean && apt-get -y autoremove && \
cp -R /usr/share/locale/en\@* /tmp/ && rm -rf /usr/share/locale/* && mv /tmp/en\@* /usr/share/locale/ && \
rm -rf /var/cache/debconf/*-old && rm -rf /var/lib/apt/lists/* && rm -rf /usr/share/doc/*
```

**Comment by tomav** [comment]
@PhilippeChepy thanks for this work!
Is there a reason you moved a lot of things into `target/` folder?
Are you sure it won't break something on running docker mail servers?

**Comment by tomav** [comment]
Hi @PhilippeChepy, let me know. Note that the branch has conflicts.

**Comment by PhilippeChepy** [comment]
I moved all files that needs to be in image in the `target/` folder just for practical reasons. I prefer doing one `COPY` instead of multiple `ADD`s in my Dockerfile. If you want to add a file in your image, you just have to add that file in the `target/` folder.
I guess it's not very important for the size reduction of the image.

**Comment by tomav** [comment]
I'm merging manually and checking things.

**Comment by tomav** [comment]
This PR is conflicting again. @PhilippeChepy could you have a look?
Thank you.

**Comment by tomav** [comment]
Closed due to inactivity.

**Pull request closed without merging**